### PR TITLE
Add partial class

### PR DESCRIPTION
**Pull request by MarianRoz**

### CODE REVIEW

File: CodeWars/Tasks/ReturnMaximumHourMinute.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@ namespace CodeWars.Tasks
     /// https://www.codewars.com/kata/62a4abf03236976b7acf7781
     /// </summary>
     /// Add string
-    public class ReturnMaximumHourMinute
+    public partial class Tasks
     {
         static Dictionary<int, int> getFrequencyMap(int[] inputArray)
```
Comment (bohdansydor) [inline CodeWars/Tasks/ReturnMaximumHourMinute.cs]: Lets use Pascal Case for method name. Here and in other places

Comment (bohdansydor) [inline CodeWars/Tasks/ReturnMaximumHourMinute.cs]: What access modifier have this method?
Let's set it explicitly

File: CodeWars/Tasks/ReturnMaximumHourMinute.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@ namespace CodeWars.Tasks
     /// https://www.codewars.com/kata/62a4abf03236976b7acf7781
     /// </summary>
     /// Add string
-    public class ReturnMaximumHourMinute
+    public partial class Tasks
     {
```
Comment (bohdansydor) [inline CodeWars/Tasks/ReturnMaximumHourMinute.cs:10]: Let's rename hashMap to be more descriptive.

File: CodeWars/Tasks/GeometricProgressionSequence.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+{
+    /// <summary>
+    /// https://www.codewars.com/kata/55caef80d691f65cb6000040/train/csharp
+    /// </summary>
+    public partial class Tasks
+    {
+        public static string GeometricSequenceElements(int a, int r, int n)
+        {
+            return string.Join(", ", Enumerable.Range(0, n).Select(i => Math.Pow(r, i) * a));
```
Comment (bohdansydor) [inline CodeWars/Tasks/GeometricProgressionSequence.cs:14]: What is string.Join, Enumerable, Enumerable.Range(0, n), Select, Math.Pow?

File: CodeWars/Tasks/CatsAndShelf.cs [hunk 4]
```diff
@@ -0,0 +1,46 @@
+﻿using System;
+namespace CodeWars.Tasks
+{
+    /// <summary>
+    /// https://www.codewars.com/kata/62c93765cef6f10030dfa92b/csharp
+    /// </summary>
+    public static partial class Tasks
+    {
+        public static int Cats(int start, int finish)
```
Comment (bohdansydor) [inline CodeWars/Tasks/CatsAndShelf.cs:9]: I think it is interesting task. Could you please implement one more solution for this task?(Don't copy solution from www.codewars.com)

File: CodeWars/Tasks/BubbleSortOnce.cs [hunk 5]
```diff
@@ -4,7 +4,7 @@ namespace CodeWars.Tasks
     /// <summary>
     /// https://www.codewars.com/kata/56b97b776ffcea598a0006f2/train/csharp
     /// </summary>
-    public static class BubbleSortOnce
+    public static partial class Tasks
     {
         public static int[] BubbleSortOnceResolve(int[] input)
```
Comment (bohdansydor) [inline CodeWars/Tasks/BubbleSortOnce.cs]: Looks like this is Bubble Sort Algorithm but not BubbleSortOnceResolve. Please doublecheck.

File: CodeWars/Tasks/ReturnMaximumHourMinute.cs (hunk 1 above)
Comment (bohdansydor) [reply CodeWars/Tasks/ReturnMaximumHourMinute.cs]: Here and in other places

Comment (MarianRoz) [reply CodeWars/Tasks/ReturnMaximumHourMinute.cs]: Done.
In a branch "feature/addTask" changed.

Comment (MarianRoz) [reply CodeWars/Tasks/ReturnMaximumHourMinute.cs]: done

Comment (MarianRoz) [reply CodeWars/Tasks/ReturnMaximumHourMinute.cs]: Also added an access modifier to the "Main" method

File: CodeWars/Tasks/ReturnMaximumHourMinute.cs (hunk 2 above)
Comment (MarianRoz) [reply CodeWars/Tasks/ReturnMaximumHourMinute.cs:10]: Renamed hashMap to NumberOfElements

Comment (MarianRoz) [reply CodeWars/Tasks/ReturnMaximumHourMinute.cs:10]: Also added the "CodeWars.sln" file

File: CodeWars/Tasks/GeometricProgressionSequence.cs (hunk 3 above)
Comment (MarianRoz) [reply CodeWars/Tasks/GeometricProgressionSequence.cs:14]: Join - сonnects elements. puts a separator between elements.
Enumerable - lists what is specified after .
Enumerable.Range(0, n) - lists in a certain range of numbers.
Select - element transformation.
Math.Pow - defines a number in power.

File: CodeWars/Tasks/CatsAndShelf.cs (hunk 4 above)
Comment (MarianRoz) [reply CodeWars/Tasks/CatsAndShelf.cs:9]: okey

File: CodeWars/Tasks/BubbleSortOnce.cs (hunk 5 above)
Comment (MarianRoz) [reply CodeWars/Tasks/BubbleSortOnce.cs]: Oh yes, you right. We overachieved this task, and I forgot about it)).

File: CodeWars/Tasks/CatsAndShelf.cs (hunk 4 above)
Comment (MarianRoz) [reply CodeWars/Tasks/CatsAndShelf.cs:9]: Added another solution for CatsAndShelf.cs named CatsAndShelfTwoTask.

Comment (MarianRoz) [reply CodeWars/Tasks/CatsAndShelf.cs:9]: https://github.com/MarianRoz/CodeWars/blob/feature/addPartial/CodeWars/Tasks/CatsAndShelfTwoTask.cs

File: CodeWars/Tasks/ReturnMaximumHourMinute.cs [hunk 6]
```diff
@@ -10,26 +10,26 @@ public partial class Tasks
...
         public static Dictionary<int, int> getFrequencyMap(int[] inputArray)
         {
-            Dictionary<int, int> hashMap = new Dictionary<int, int>();
+            Dictionary<int, int> NumberOfElements = new Dictionary<int, int>();
```
Comment (bohdansydor) [inline CodeWars/Tasks/ReturnMaximumHourMinute.cs]: https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md

Please, chage acording to Naming Conventions

File: CodeWars/Program.cs [hunk 7]
```diff
@@ -1,19 +1,35 @@
 ﻿using System;
 using CodeWars.Tasks;
 
-namespace BubblesortOnce
+namespace CodeWars.Tasks
 {
-    class Program
```
Comment (bohdansydor) [inline CodeWars/Program.cs:6]: Actually you do not need to change class Program

File: CodeWars/Tasks/BubbleSortAlgorithm.cs [hunk 8]
```diff
@@ -4,9 +4,9 @@ namespace CodeWars.Tasks
     /// <summary>
     /// https://www.codewars.com/kata/56b97b776ffcea598a0006f2/train/csharp
```
Comment (bohdansydor) [inline CodeWars/Tasks/BubbleSortAlgorithm.cs]: Nice try)))
Lets create separete file for BubbleSortOnceResolve and BubbleSortAlgorithm. 
Please, remove codewars link as it is not refer to BubbleSortAlgorithm.

File: CodeWars/Tasks/CatsAndShelfTwoTask.cs [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+namespace CodeWars.Tasks
+{
+    /// <summary>
+    /// https://www.codewars.com/kata/62c93765cef6f10030dfa92b/csharp
+    /// </summary>
+    public partial class Tasks
+    {
+        public static int CatsTwoTask(int start, int finish)
+        {
+            int Result;
```
Comment (bohdansydor) [inline CodeWars/Tasks/CatsAndShelfTwoTask.cs]: Wrong naming, please fix

File: CodeWars/Tasks/CatsAndShelfTwoTask.cs [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+            int Division = (finish - start) / 3;
+            int RemainderFromDivision = (finish - start) % 3;
+
+            return Result = (RemainderFromDivision == 0) ? Division : ((RemainderFromDivision == 1) ? Division += 1 : Division += 2);
```
Comment (bohdansydor) [inline CodeWars/Tasks/CatsAndShelfTwoTask.cs]: Could you please explain this row.

File: CodeWars/Tasks/CatsAndShelfTwoTask.cs [hunk 11]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+namespace CodeWars.Tasks
+{
+    /// <summary>
+    /// https://www.codewars.com/kata/62c93765cef6f10030dfa92b/csharp
+    /// </summary>
+    public partial class Tasks
+    {
+        public static int CatsTwoTask(int start, int finish)
```
Comment (bohdansydor) [inline CodeWars/Tasks/CatsAndShelfTwoTask.cs]: Could you please explain idea of this solution.

File: CodeWars/Tasks/TakeATenMinutesWalk.cs [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+namespace CodeWars.Tasks
+{
+    /// <summary>
+    /// https://www.codewars.com/kata/54da539698b8a2ad76000228
+    /// </summary>
+    public partial class Tasks
+    {
+        public static bool IsValidWalk(string[] walk)
+        {
+            int n=0, s=0, w=0, e=0;
```
Comment (bohdansydor) [inline CodeWars/Tasks/TakeATenMinutesWalk.cs:11]: What do you think these letters mean?

File: CodeWars/Tasks/ReturnMaximumHourMinute.cs (hunk 6 above)
Comment (MarianRoz) [reply CodeWars/Tasks/ReturnMaximumHourMinute.cs]: Renamed "NumberOfElements" to "numberOfElements"

**Review by bohdansydor — APPROVED** [review]
I like it. Great job!

**Review by bohdansydor — APPROVED** [review]
Now you can merge this branch

**Pull request merged**